content/en/blog/sign-verify-plugins-rules/index.md

```diff
@@ -0,0 +1,178 @@
+
+There are cheap preventive safeguards that can be implemented in open source projects especially, where stakeholders platea can be pretty wide considering the contributions. For example branch protection rules are usually simple per-code repository configurations in providers (e.g. GitHub) and alongside pull-request based flows enforcing code review quorum, are also standard best practices nowadays. The same applies to reproducible builds, dependency pinning, build steps isolation, MFA authentication to repository providers, etc.
+
+On the other side, safeguards with an high utility value can require a considerable amount of effort to be implemented end-to-end (producer and consumer sides).
+
+MITRE and OpenSSF increased their efforts to improve SSC security recently in general and in the open source, by providing frameworks and tools that increase the utility to cost ratio.
+MITRE for example proposes an end-to-end framework to preserve the integrity of the software supply chain. OpenSSF develops the SLSA framework that groups several security best-practices for open source projects.
+
+Moreover, the in-toto Attestation Framework defines a fixed, lightweight Statement that communicates some information about the execution of the software supply chain, such as if the source was reviewed, or if it went through a SLSA conformant build process.
+
+Supply chain security is important to be constantly increased through safeguards placed as soon as possible in the supply chain ("shift left") and needs to be maintained in a continuous manner reducing the operational burden (DevSecOps) because technologies evolve so attacks do.
+ 
+Software artifacts signatures are one of the safeguards with a reasonable amount of effort nowadays, thanks also to important projects like Sigstore's [cosign](https://github.com/sigstore/cosign) for artifacts distributed as container images. So this was one of the goals decided for the Falco Software Supply Chain Working Group.
+
+## Artifacts distributed as OCI
+
+Since January 2023 we started distributing Falco plugins and rules as [OCI artifacts](https://github.com/oras-project/artifacts-spec), in addition to other ways. If you are not familiar with the concept, think about OCI artifacts as an unit of data that can be tagged, stored and indexed in container registries alongside your container images but can hold any type of file. You can find the source and the data for Falco's artifacts in the [plugins](https://github.com/falcosecurity/plugins) and [rules](https://github.com/falcosecurity/rules) repositories respectively. We are using GitHub Container Registry to publish them so you can find them in the "Package" section of each repository.
```

Comment:
```suggestion
Since January 2023 we started distributing Falco plugins and rules as [OCI artifacts](https://github.com/oras-project/artifacts-spec), in addition to other ways. If you are not familiar with the concept, think about OCI artifacts as a content-addressable unit of data that can be tagged, stored and indexed in container registries alongside your container images but can hold any type of file. You can find the source and the data for Falco's artifacts in the [plugins](https://github.com/falcosecurity/plugins) and [rules](https://github.com/falcosecurity/rules) repositories respectively. We are using GitHub Container Registry to publish them so you can find them in the "Package" section of each repository.
```